README.md

```diff
@@ -1,31 +1,198 @@
-
-python3 ./beacon_simulator.py -ip 192.168.0.5 -p 2000 -i 10 -j 3 -m 1024 <br/>
-python3 ./beacon_simulator.py -ip 192.168.0.5 -p 2000 --interval 120 --jitter 12 --max_payload 1024 --tcp <br/>
-On your client device: python3 ./tcp_client.py <br/>
-On your mock C2 server device: python3 ./tcp_server.py <br/>
-
-For using the client/server python scripts they work in pairs. The UDP
-Client script works with the UDP Server script and the TCP Client script
-works with the TCP Server script. The client/server scripts require some
-manual configuration within the scripts. You will have to put the
-destination IP(s) at “server = [your.server.goes.here]”. The port that
-the scripts are running on by default is 9000 but it can be changed at
-the line with “PORT = 9000” or “SERVER_PORT = 9000”. This script should
-scale to as many destinations as you would like to have. Each script has
-a printout of each message that is sent, received, and counts the number
-of beacons that have currently been sent out. By default the number of
-bytes sent for each beacon is a random number between 0 and 1200. This
-can be changed at the line with “m = random.randint(0,1200)”. By default
-the beaconing interval is between 30 and 60 seconds. This can be changed
-at the line with “rsleep = random.randint(30,60)”. The client/server
-scripts have a comment on most of the lines with a brief description of
-its purpose if you are unsure of it.
+<h1 align="center">threat-tools</h1>
+
+This repository contains several scripts that will allow you to mimic a compromised system calling home to a command and control (C2) server. None of the tools exfiltrate any data and are designed to test your environment's ability to detect a wide range of C2 channels.
+
+## Table of Contents
+[Overview](#overview)</br>
+[Dependencies](#dependencies)</br>
+[`simple-listener.sh`](#simple-listenersh)</br>
+[`beacon-simulator.py`](#beacon-simulatorpy)</br>
+[`beacon-simulator.sh`](#beacon-simulatorsh)</br>
+[`simple-beacon.sh`](#simple-beaconsh)</br>
+[Python Script Pairs](#python-script-pairs)</br>
+[Running Tools in the Background with `screen`](#running-tools-in-the-background-with-screen)
+
+
+## Overview
+In order to run these scripts, you will need two systems: one to act as the compromised client sending the beacon, and one to act as the C2 server. The client will need to run one of the beacon simulator scripts, and the server will need to run one of the scripts to set up a listener. With exception to the scripts within `python-scripts`, all beacon simulators are meant to be used in tandem with `simple-listener.sh` running on the server. The only exception to this is if the client is running `simple-beacon.sh`, which can be used on its own without additional scripts running on the server.
```

Comment:
Let's be more specific with how simple-beacon is different. Let's go with something along the lines of 

The only exception to this is if the client is running `simple-beacon.sh`, which can simulate beacon traffic by repeatedly sending web requests to any public web server. 

We may want to change the defaults in this script to point it at activecountermeasures.com by default.

Reply:
Done and done! Also updated its section of the README to include this specificity.

I have made some minor tweaks to the scripts themselves, but it was purely editing to make sure comments and things were specific. I updated the PR to include these changes.